### PR TITLE
Updated the commerce upgrade docs

### PR DESCRIPTION
**Pull request by jonat123**
## Description
Updated the docs so that they are more in line with how we have structured the documentation for our other DXP products.

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

Umbraco Commerce ,10 and 12

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

**Review by mattbrailsford — CHANGES_REQUESTED** [review]
Content seems fine, but it would probably now make sense to move the "Version Specific Upgrade Notes" under the "Upgrading" heading, rather than under "Installation"

**Comment by jonat123** [comment]
@mattbrailsford Ahhh, that is a good point!

I'll move it :-)

File: 12/umbraco-commerce/getting-started/upgrading.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Upgrading Umbraco Commerce
+
+This article shows how to manually upgrade Umbraco Commerce to run the latest version.
+When upgrading Umbraco Commerce, be sure to also consult the [version specific upgrade](/112/umbraco-commerce/getting-started/upgrading/version-specific-upgrades.md) notes to learn about potential breaking changes and common pitfalls.
```
Comment (alina-tincas) [inline 12/umbraco-commerce/getting-started/upgrading.md:4]:
```suggestion
When upgrading Umbraco Commerce, be sure to also consult the [version specific upgrade](/12/umbraco-commerce/getting-started/upgrading/version-specific-upgrades.md) notes to learn about potential breaking changes and common pitfalls.
```

File: 12/umbraco-commerce/getting-started/upgrading/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Upgrading Umbraco Commerce
+
+This article shows how to manually upgrade Umbraco Commerce to run the latest version.
+When upgrading Umbraco Commerce, be sure to also consult the [version specific upgrade](/10/umbraco-commerce/getting-started/installation/version-specific-upgrades.md) notes to learn about potential breaking changes and common pitfalls.
```
Comment (alina-tincas) [inline 12/umbraco-commerce/getting-started/upgrading/README.md:4]:
```suggestion
When upgrading Umbraco Commerce, be sure to also consult the [version specific upgrade](10/umbraco-commerce/getting-started/upgrading/version-specific-upgrades.md) notes to learn about potential breaking changes and common pitfalls.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Looks good, I have only one suggestions to resolve the links to reference to the correct path 😁

**Pull request merged**